html/php-pre-challenge3/index.php

```diff
@@ -3,21 +3,28 @@
 $limit = mb_convert_kana($limit, 'a' , 'UTF-8');
 if(!is_numeric($limit) || $limit <= 0 || preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit)) {
     http_response_code(400);
+    echo json_encode('invalid limit : ' . $limit) . '<br>' . '1以上の整数を入力してください。';
```

Comment:
json_encodeの外に値を繋げるとjsonじゃなくなってしまいます。
今回のレスポンスはjson形式に統一したいので、レスポンスの値はjson_encodeの中に入れてください。

```suggestion
    echo json_encode('invalid limit : ' . $limit);
```